src/Sql/dbo/Stored Procedures/Organization_Search.sql

```diff
@@ -1,14 +1,14 @@
 ﻿CREATE PROCEDURE [dbo].[Organization_Search]
     @Name NVARCHAR(50),
-    @UserEmail NVARCHAR(50),
+    @UserEmail NVARCHAR(256),
     @Paid BIT,
     @Skip INT = 0,
     @Take INT = 25
 WITH RECOMPILE
 AS
 BEGIN
     SET NOCOUNT ON
-    DECLARE @NameLikeSearch NVARCHAR(55) = '%' + @Name + '%'
+    DECLARE @NameLikeSearch NVARCHAR(261) = '%' + @Name + '%'
```

Comment:
Why is this being changed here? No changes were being made to `@Name` and `@UserEmail` isn't being appended to it?

Reply:
Thank you very much for the review. Most probably assumed it would have to change as in User_search and didn't pay attention to the variables. Thanks for catching that.